.eslintrc.js

```diff
@@ -1,19 +1,8 @@
 module.exports = {
-  env: {
-    es6: true,
-    'shared-node-browser': true,
-  },
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: './tsconfig.json',
-    sourceType: 'module',
-    ecmaVersion: 2018,
-    ecmaFeatures: {
-      jsx: true,
-    },
-  },
-  settings: {
-    'import/ignore': ['node_modules'],
+  extends: '../shared-configs/eslint-lib',
+  rules: {
+    'no-restricted-imports': ['error', { patterns: ['./*', '../*'] }],
+    'func-style': ['error', 'expression'],
```

Comment:
This one is for `const myFunc = ...` instead of `function myFunc()...`. Just kept it because it was here already (in addition to the base config that was extracted).